tests/ha/single_pod_restart/test_data_server_pod_restart.py

```diff
@@ -518,3 +524,450 @@ def test_io_data_server_pod_restart(self):
+            resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                        log_prefix=self.test_prefix_deg,
+                                                        skipcleanup=True, setup_s3bench=False,
+                                                        nsamples=5, nclients=5)
+            assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("STEP 4: Create new objects and perform WRITEs/READs/Verify with variable "
+                    "object sizes in degraded mode")
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix,
+                                                    skipcleanup=True, setup_s3bench=False,
+                                                    nsamples=5, nclients=5)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("STEP 4: Performed WRITEs/READs/Verify with variable sizes objects in "
+                    "degraded mode")
+
+        LOGGER.info("STEP 5: Restore data pod and check cluster status.")
+        pod = const.POD_NAME_PREFIX
+        self.restore_method = self.pod_dict.get(pod)[-1]
+        resp = self.ha_obj.restore_pod(pod_obj=self.node_master_list[0],
+                                       restore_method=self.restore_method,
+                                       restore_params={
+                                           "deployment_name": self.pod_dict.get(pod)[-2],
+                                           "deployment_backup": self.deployment_backup,
+                                           "num_replica": self.pod_dict.get(pod)[2],
+                                           "set_name": self.pod_dict.get(pod)[1]})
+        LOGGER.debug("Response: %s", resp)
+        assert_utils.assert_true(resp[0], f"Failed to restore pod by {self.restore_method} way")
+        LOGGER.info("Successfully restored pod by %s way", self.restore_method)
+        LOGGER.info("STEP 5: Successfully started data pod and cluster is online.")
+        self.restore_pod = False
+
+        LOGGER.info("STEP 6: Perform READ/Verify on data written with buckets created in healthy "
+                    "mode in step-1")
+        resp = self.ha_obj.ha_s3_workload_operation(s3userinfo=list(users.values())[0],
+                                                    log_prefix=self.test_prefix, skipwrite=True,
+                                                    skipcleanup=True, setup_s3bench=False,
+                                                    nsamples=5, nclients=5)
+        assert_utils.assert_true(resp[0], resp[1])
+        LOGGER.info("STEP 6: Successfully run READ/Verify on data written with buckets created in "
+                    "healthy mode in step-1")
+
+        if CMN_CFG["dtm0_disabled"]:
+            LOGGER.info("STEP 7: Perform READs and verify DI on the data written on buckets "
```

Comment:
We can let it do write, read, verify.
Remove skip for write.

Reply:
This is read/ verify of step 4 when data is down, we have write/read/verify in next step.
So i guess here no need of any change? @apurwamohite